src/js/components/recipient/topFive/TopFive.jsx

```diff
@@ -77,12 +77,14 @@ export default class TopFive extends React.Component {
                         {rows}
                     </tbody>
                 </table>
+
+                { /* Out of current sprint scope
```

Comment:
I would either leave this in and hide it with CSS, or remove the commented out code for now.